### ISSUE TITLE
multiple sessions on same host remove namespaces that are still in use

### ISSUE THREAD

**Issue by ak-1** [jamesmcm/vopono#175]
Running a second session in parallel on the same host currently removes the namespace of the other running vopono session:

    DEBUG vopono::util > Existing namespaces: ["vopono_..."]
    DEBUG vopono::util > Removing dead namespace: vopono_...
    DEBUG vopono::util > ip netns delete vopono_...

**Comment by jamesmcm** [comment]
Could you please add more details about what that namespace is running?

In theory it uses the PID to check but this might be problematic if the process forks such that the original PID isn't running anymore.

**Comment by ak-1** [comment]
Actually the issue seems to be two sessions run by different users on the same box.

**Comment by jamesmcm** [comment]
Was this caused by the names previously being truncated at the start, which you fixed in #177 ?

**Comment by jamesmcm** [comment]
@ak-1 yeah I think that should work, in the end I'd like to avoid shelling out but it'll do for now.

**Comment by ak-1** [comment]
@jamesmcm About the relationship to #177: Good question, but I think this is unrelated.
If I understand the code correctly vopono currently only tracks other vopono executions by the same user and in my case there was another session by a different user whose namespace got removed.

**Comment by jamesmcm** [comment]
Somewhat fixed in 0.10.1 - now the cleaning of dead namespaces will leave those which have PIDs running - but this doesn't apply for the cleaning of the created network namespace when vopono is closed.

So this could still be an issue if you run two separate vopono instances on different users, to the same network namespace.
